react/src/components/Timeline/components/TimelineEntry.tsx

```diff
@@ -3,10 +3,21 @@ import TimelineBullet from "./TimelineBullet";
 
 export interface ITimelineEntryProps {
   content: string | undefined;
-  time: string | undefined;
+  time: string | Date | undefined;
 }
 
 function TimelineEntry({ content, time }: ITimelineEntryProps) {
+  function formatTime(date: Date | string): string {
```

Comment:
could you pull this out into helpers. Otherwise, this function is defined in each rerender multiple times.